packages/components/src/internal/components/editable/Controls.tsx

```diff
@@ -207,7 +206,7 @@ interface RightClickToggleProps {
     bsRole?: any;
     onClick?: any;
 }
-
+// TODO: unused, remove
```

Comment:
I don't see any usages either, so I think it can be removed. Interestingly, LKB still has an unused copy of this as well which can likely be removed.

Reply:
Good catch, I'll remove it in Biologics as well